scanpipe/management/commands/__init__.py

```diff
@@ -64,6 +64,13 @@ def get_run_status_code(self, run):
         return status
 
 
+def indent(text, by=0, sep="\n"):
+    """
+    Indent each line of the provided `text`.
+    """
+    return sep.join(" " * by + line for line in text.split(sep))
```

Comment:
Could this be a job for the stdlib textwrap? https://docs.python.org/3/library/textwrap.html

Reply:
Yes, thanks for the tip.